### PR TITLE
Add "Thrown 80 Feet"

### PR DESCRIPTION
**Pull request by SpartanCPA**

### CODE REVIEW

**Comment by stwlam** [comment]
Could you edit src/module/item/weapon/values.ts as well?

**Review by stwlam — CHANGES_REQUESTED** [review]
Could you edit src/module/item/weapon/values.ts as well?

**Comment by SpartanCPA** [comment]
> Could you edit src/module/item/weapon/values.ts as well?

Done

File: src/module/item/weapon/values.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ const WEAPON_PROPERTY_RUNE_TYPES = new Set([
     "wounding",
 ] as const);
 
-const THROWN_RANGES = new Set([10, 15, 20, 30, 40, 60, 100] as const);
+const THROWN_RANGES = new Set([10, 15, 20, 30, 40, 60, 80, 100] as const);
 const WEAPON_RANGES = new Set([...THROWN_RANGES, 50, 70, 80, 90, 120, 140, 150, 180, 240, 300] as const);
```
Comment (stwlam) [inline src/module/item/weapon/values.ts:94]:
```suggestion
const WEAPON_RANGES = new Set([...THROWN_RANGES, 50, 70, 90, 120, 140, 150, 180, 240, 300] as const);
```

**Review by stwlam — APPROVED** [review]

**Pull request merged**